packages/smart-vaults/src/vault/SmartVault.sol

```diff
@@ -37,11 +37,22 @@ contract SmartVault is IAccount, Ownable, UUPSUpgradeable, MultiSignerAuth, ERC1
 
     /// @notice Single User Op Signature Scheme.
     struct SingleUserOpSignature {
+        /// @notice Upper limit for Max Priority Fee Per Gas that should be charged by the userOp. This is included in
+        /// the light userOp hash to ensure last signer does not exceed the specified gas price. Can be left empty if
```

Comment:
nit: i might be getting picky here, but when you say can be left empty what does that mean? Won't a value always be needed here? It's just if threshold = 1 then we won't use it, so doesn't matter what it's set to. But like...by empty you mean `0`, right?

Again, just making sure I'm understanding how all this works.

Reply:
yes by empty I mean 0, sorry should have made it clear